arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -252,7 +293,48 @@
           "200": {
             "description": "OK",
             "schema": {
-              "$ref": "#/definitions/ResourceList[RegistryParameters]"
+              "$ref": "#/definitions/ResourceList[Registry]"
+            }
+          }
+        },
+        "deprecated": false

```

Comment:
Please add 
"x-ms-pageable": {
    "nextLinkName": null
}
to Registries_List and Registries_ListByResourceGroup.